src/components/SearchGrid.vue

```diff
@@ -168,6 +168,7 @@ export default {
 
   .search-grid_analytics h5,
   .search-grid_layout-control h5 {
+    padding-top: 13px;
```

Comment:
Could you use `vh` instead of fixed `px`?

Reply:
@brenoferreira Can I complete this task?

Reply:
I am just doing it now!

Reply:
![Screenshot from 2019-03-27 02-16-48](https://user-images.githubusercontent.com/36623306/55032383-c2e8bf00-5036-11e9-92b5-2ddcafa2438f.png)
![Screenshot from 2019-03-27 02-11-16](https://user-images.githubusercontent.com/36623306/55032393-c714dc80-5036-11e9-86e1-138a41a1e572.png)
Before and After photos!

Reply:
Hey @brenoferreira could u have a look at this!